### ISSUE TITLE
UI: Search by government entity

### ISSUE THREAD

**Issue by bibAtWork** [CMS-Garden/ftm#11]
Scope: Development of a webpage to search records by government entity properties.

Note:
- Focus should be on a high usability and accessibility.
